Zend/README.md

```diff
@@ -125,15 +125,14 @@ Executor's code is generated by PHP script zend_vm_gen.php it uses
 definitions. It is included from `zend_compile.h`. The second one is an executor
 code itself. It is included from `zend_execute.c`.
 
-`zend_vm_gen.php` can produce different kind of executors. You can select
-different opcode threading model using `--with-vm-kind=CALL|SWITCH|GOTO`. You
-can disable opcode specialization using `--without-specializer`. You can include
-or exclude old executor together with specialized one using
-`--without-old-executor`. At last you can debug executor using original
-`zend_vm_def.h` or generated file `zend_vm_execute.h`. Debugging with original
-file requires `--with-lines` option. By default ZE2 uses the following command
-to generate executor:
+`zend_vm_gen.php` can produce different kind of executors. You can select a
+different opcode threading model using `--with-vm-kind=CALL|SWITCH|GOTO|HYBRID`.
+You can disable opcode specialization using `--without-specializer`.
+At last you can debug the executor using the original `zend_vm_def.h` or the generated
+`zend_vm_execute.h` file. Debugging with the original file requires the `--with-lines` option.
+By default ZE3 (since PHP 7) uses the following command to generate executor:
```

Comment:
As of PHP 8.0.0, [it is ZE4](https://github.com/php/php-src/blob/master/Zend/zend.h#L23). Might be better to omit the version, and just say "Zend Engine".

Reply:
@cmb69 I can make a new patch to update README.md as your comment.
However, in the zend.h, I found another macro #define ZEND_ENGINE_3 defined immediately following ZEND_VERSION "4.2.0-dev", should we also update it to ZEND_ENGINE_4? (A rough search in source code shows no reference to this macro definition. )
[Zend Engine 3 git commit](https://github.com/php/php-src/commit/90d916c4b10c1a94adc89d3295260dbbfb50a778#diff-49d1aeab0b09a6f9d475cd503187847f2d930989bfe729496a949e574d067649 
) upgraded both ZEND_VERSION and ZEND_ENGINE_3.

Reply:
> 

New README update PR is here: https://github.com/php/php-src/pull/8057

Reply:
I don't think we can replace `ZEND_ENGINE_3`, because it is likely used by some extensions. Updating to `ZEND_ENGINE_4` makes sense for PHP 9, if there are sufficient API changes (PHP 7 and PHP 8 didn't differ that much regarding ZE).

Reply:
got it. Thanks @cmb69 for the clarification.